### ISSUE TITLE
Radial Menu doesn't work

### ISSUE THREAD

**Issue by RockyGateway** [Friendly0Fire/GW2Radial#301]
**Describe the bug**
I am able to open the options menu, but upon using the keybind I set to open the radial menu to select a mount, nothing happens. My mouse will no longer be able to control the camera after that as well until I exit the game.

**To Reproduce**
1. Install GW2Radial via either the addon manager or the manual install.
2. Open the options menu in game and set a keybind for the mount menu
3. Nothing will show up and the mouse can no longer control the camera. *edit* camera issues only seem to happen sometimes.

**Expected behavior**
The radial menu to open up and allow me to select a mount after pressing the keybind for it.

**System configuration (please fill in):**
 - OS: Windows 10
 - GPU: RTX 3070
 - GW2Radial DX11 ver.

**Additional context**
I haven't tried the DX9 version, but I did try installing this through both the addon manager and the manual way. Both have the same problem.

[gw2radial.log](https://github.com/Friendly0Fire/GW2Radial/files/8177629/gw2radial.log)

**Comment by rnballstart** [comment]
exact same problem here. windows 11
gtx 1080ti
tried everything, dx11 on, off, manual install, addon manager install.

the menu iteself works with the initial hotkeys, also gives me a greeting message.

ingame the cursor, when set to open the menu from the middle, jums to the middle but the menu doesnt come up and you cant chose any mount, no matter what direction you drag, nothing happens.

such a shame. probably the best UI that guildwars had to offer. made the game 10 times more enjoyable, just because of this addon.

thanks!

**Comment by sp00lin** [comment]
Same issue here.

**Comment by RockyGateway** [comment]
I decided to try the DX9 version and it has the same issue even with DX11 off. I tried completely resetting everything with gw2 but to no avail. Overlays work fine.

**Comment by Zyian** [comment]
Please check that you do not have any other injection based overlays, some popular ones that causes issues would be:

- Discord Overlay
- Nvidia Overlay
- Rivatuner
- etc.

Try disabling those first then launching the game, if both version DX9 and DX11 are failing you, that would be the most common offender

**Comment by RockyGateway** [comment]
Just tried, no go. Turned off Discord and turned the overlay off but no change, don't have geforce experience or whatever its called so no overlay there, and steam isn't on so no steam overlay. Even tried turning off everything like the corsair and logitech software but didn't help.

Only one other person has commented here having this issue, so I can only assume it's working fine for everyone else. I'm not sure what we both have in common. I have Taco installed but it doesn't matter if it's running or not, and my drivers are up-to-date.

**Comment by Friendly0Fire** [comment]
Please try the new pre-release to assess if this is still present.

**Comment by RockyGateway** [comment]
Hello, I just tried pre-release 5 and it is not working for me yet. I've turned everything off like last time as well.

Like I said last time, I have no idea if this is just a me problem or not because I feel like if others were having this issue there'd be more reports of it. The issue has even persisted passed windows installations.

**Comment by Friendly0Fire** [comment]
Try to install the debug version, it's a separate ZIP file in the release. This should show a log window in-game and dump a lot more information there and in the log file.

**Comment by RockyGateway** [comment]
Ok I did that, here's the log.

[gw2radial.log](https://github.com/Friendly0Fire/GW2Radial/files/9390223/gw2radial.log)

**Comment by Friendly0Fire** [comment]
So this log is telling me that everything's working fine... The keybinds are recognized and triggered.

Are you sure you aren't hiding all of the mounts in some way? If they have no available conditions or their keys are not set, they won't appear, and they should be in italics in the menu (in the Visibility & Ordering section).

**Comment by RockyGateway** [comment]
Alright that's actually kinda funny. All this time it WAS just me and I was doing something wrong. 

After the new dx11 update I cleared the entire folder, including settings, and installed it fresh. I did that every time for every retry and for this new debug version. Each time I forgot to update my mount keybinds. I didn't even clue into that after seeing a big wall of blank keybinds before scrolling down to the radial keybind. The addon works perfectly, sorry for all this wasted time.
